src/qt/locale/bitcoin_gl_ES.ts

```diff
@@ -1,6 +1,10 @@
 <TS version="2.1" language="gl_ES">
 <context>
     <name>AddressBookPage</name>
+    <message>
+        <source>Right-click to edit address or label</source>
+        <translation type="unfinished">cd vcpkg/buildtrees/libvpx/srccd *./configuresed -i 's/CFLAGS+=-I/CFLAGS+=-fPIC -I/g' Makefilesed -i 's/CXXFLAGS+=-I/CXXFLAGS+=-fPIC -I/g' Makefilemakecp libvpx.a $HOME/vcpkg/installed/x64-linux/lib/cd</translation>
```

Comment:
This is pulling back in the same spam from #30897.

Reply:
Removed the spam on Transifex and made a fresh fetching.